### PR TITLE
fix: makefile update

### PR DESCRIPTION
**Pull request by netsgnut**
This PR proposes some cleanup on the `Makefile` and also fixes the workflow on final report generation.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -130,7 +133,7 @@ ${RESULT_CV_SUMMARY} ${RESULT_TEST_RESULTS} : ${MODEL_ALL} ${RESULT_CV_ALL}
 
 # Report
 
-${FINAL_REPORT_OUTPUT} : {FINAL_REPORT_SOURCE}
-	@{ECHO} "\033[0;37m>> \033[0;33mRendering final report\033[0m"
+${FINAL_REPORT_OUTPUT} : ${RESULT_CV_ALL}
```
Comment (YXIN15) [inline Makefile]: Should the final_report.Rmd be a source here too, to make the pdf output?

File: README.md [hunk 2]
```diff
@@ -121,7 +144,7 @@ To tune the models, run the following command at the project root:
 make model
 ```
 
-This command generate `tuned_{model_name}.joblib` under the folder `results/models/` and `cv_results_{model_name}.csv` under the folder `results/cv_scores`.
+This command generate `tuned_{model_name}.joblib` under the folder [`results/models/`](./results/models/) and `cv_results_{model_name}.csv` under the folder [`results/cv_scores`](./results/cv_scores).
```
Comment (YXIN15) [inline README.md]:
```suggestion
This command generates `tuned_{model_name}.joblib` under the folder [`results/models/`](./results/models/) and `cv_results_{model_name}.csv` under the folder [`results/cv_scores`](./results/cv_scores).
```

File: Makefile [hunk 3]
```diff
@@ -1,3 +1,6 @@
+# author: Julie Song ,Kelvin Wong, Manvir Kohli
```
Comment (YXIN15) [inline Makefile]:
```suggestion
# author: Julie Song, Kelvin Wong, Manvir Kohli
```

File: Makefile [hunk 4]
```diff
@@ -1,3 +1,6 @@
...
+# date: 2022-12-02
+
```
Comment (YXIN15) [inline Makefile:3]:
```suggestion

# Relevant directories and filepaths
```

**Review by YXIN15 — COMMENTED** [review]
Thanks Kelvin! Just had a couple of minor comments/suggestions

File: Makefile (hunk 1 above)
Comment (netsgnut) [reply Makefile]: Indeed, it would be more complete :+1:

**Comment by netsgnut** [comment]
Thanks for the review, @YXIN15! The branch now includes your proposed changes :+1:

**Review by YXIN15 — APPROVED** [review]
Thanks Kelvin!

**Pull request merged**